### DISCUSSION TITLE
Some suggestons (medals and mods)

### DISCUSSION THREAD

**Discussion by lucalima99** [ppy/osu#24198] [Ideas]
Medals:
With the medals bering fun to do and a community based on found medals i always thought if w was community could make medals and like if peaple realy like it it could be offical
Like via Profile where they have Custom Medals to chalange yourself 
I had some in mind
Rigged: get 777 on mod customisation or in gameplay on random mod on jackpot sad girl (or anything relatedon money) and clear it (extra mods allowed)
Taste It!: D0 7 maps based on ranbow colors on or any order (example: Red Zone > Yellow Splach etc.)
Mods:
I thought of some mod idea 
diffculty reduction:
Hold (Trasforms sliders into osustream! hold notes, thinking the note stays on slider start, middle or the end, maybe as a mod customisation)
Slideless (Transforms Sliders and reverse arrow(s) into normal circles, reverse arrow)
Maze (Everything is sliders mod customisation if 1 2 becomes sliders or each combo is a slider but the original slider with stays the same)
Difficulty Increase:
Mastermind: (the game is trying to mess you up) (Basicy imagine Wario Interrupts https://www.youtube.com/watch?v=0Q1EQGPlbsk when you trying to play the map)
Fun:
Endless (Go utill Ya Drop) (On the End of a Beatmap you go for a other one with a similar Diffculty or higher)
Artist (This is my magnum opus) (Paint the scream BG based on the combo color)

**Comment by peppy** [comment]
cc @Ephemeralis
